WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1085,16 +1094,25 @@ extension OrderDetailsDataSource {
                 }
                 headerStyle = .actionablePrimary(actionConfig: headerActionConfig)
             } else {
-                headerStyle = .primary
+                headerStyle = .twoColumn
```

Comment:
Probably not for this PR or necessarily even you to do, but this grabbed my interest. It seems we don't use the `rightTitle` for section headers any more, so don't need a twoColumn layout anywhere in this screen. I don't know if you tried using a more built in one?

Reply:
Good point @joshheald , I create an issue to address that in the future https://github.com/woocommerce/woocommerce-ios/issues/11113